spec/features/borrower_creates_loan_request.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.feature "borrower creates loan request" do
+  let(:borrower) { User.create(name: "Richard",
+                               email: "richard@example.com",
+                               password: "password",
+                               role: "borrower")
+                   }
```

Comment:
`end` at 8, 19 is not aligned with `let(:borrower) { User.create(name: "Richard",` at 4, 2